### PR TITLE
Component/Sidebar Changes

### PR DESCRIPTION
**Pull request by ashu75575**
- Add Space selector dropdown component
-  added searchbar on sidebar component and remove the icon according to figma
- Made changes to styling of sidebar
-

### CODE REVIEW

File: studio/src/actions/spaces.js
Comment (vsumit89) [inline studio/src/actions/spaces.js:1]: remove the console.log

File: studio/src/components/GlobalNav/Sidebar.js [hunk 1]
```diff
@@ -136,16 +140,38 @@ function Sidebar({ superOrg, permission, orgs, loading, applications, services,
       <div
         style={{
           display: 'flex',
-          flexDirection: collapsed ? 'column' : 'row',
+          flexDirection: 'column',
```
Comment (vsumit89) [inline studio/src/components/GlobalNav/Sidebar.js:155]: can you explain this change?

File: studio/src/actions/spaces.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ export const addSpace = (data) => {
 };
 
 export const deleteSpace = (id) => {
+  console.log('deleteSpace', id);
```
Comment (vsumit89) [inline studio/src/actions/spaces.js]: remove console.log

**Review by vsumit89 — CHANGES_REQUESTED** [review]

**Review by vsumit89 — DISMISSED** [review]

File: studio/src/components/GlobalNav/Sidebar.js (hunk 1 above)
Comment (ashu75575) [reply studio/src/components/GlobalNav/Sidebar.js:155]: its is related searchbar and spaceselector button allignment to make the change the flex direction to be in column only for both cases
figma-
image

### before change
image

**Review by surajmn1 — CHANGES_REQUESTED** [review]
@tejveer777  please include only studio files and remove changes in docker-compose.yml , readme.md , volumes folders .

**Pull request merged**